src/ckb/channel.rs

```diff
@@ -1854,22 +1861,21 @@ impl ChannelActorState {
             partial_signatures,
         )?;
 
-        let witness = self.create_witness_for_funding_cell(signature, None);
```

Comment:
The fix is essentially here. Other code change is to make it easier to debug.

Reply:
suggest to change the `version` parameter from `Option<u64>` to `u64` since we need to pass a non-empty version in all case.